plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -333,11 +334,11 @@ protected static Optional<ColumnMapping> mapToUnboundedVarchar(JdbcTypeHandle ty
         return Optional.of(ColumnMapping.sliceMapping(
                 unboundedVarcharType,
                 varcharReadFunction(unboundedVarcharType),
-                (statement, index, value) -> {
+                SliceWriteFunction.of(Types.NULL, (statement, index, value) -> {
```

Comment:
what's the advantage of `statement.setNull(index, Types.NULL)` over `statement.setObject(index, null);`?
please document

Reply:
It's the same but explicit

Reply:
So this is a "no change"? please revert.

Reply:
Done.